app-emulation/lxd/files/lxd-0.18.confd

```diff
@@ -0,0 +1,31 @@
+LXD_OPTIONS=""
+
+# Group which owns the shared socket
+LXD_OPTIONS="${LXD_OPTIONS} --group lxd"
```

Comment:
Good call.  I guess I didn't know if += was a bashism, but it's not.  

Init scripts, unlike ebuilds, are expected to be portable shell, yes?  I read that somewhere.